### PR TITLE
[#496] Add audio content model

### PR DESCRIPTION
**Pull request by paulodiniz**

### CODE REVIEW

File: lib/java/mapping/src/main/java/co/airy/mapping/sources/twilio/TwilioMapper.java [hunk 1]
```diff
@@ -29,16 +30,31 @@
...
-
         final String mediaUrl = decodedPayload.get("MediaUrl");
+
         if (mediaUrl != null && !mediaUrl.isBlank()) {
-            contents.add(new Image(mediaUrl));
+            if(isImage(mediaUrl)) {
+                contents.add(new Text(decodedPayload.get("Body")));
+                contents.add(new Image(mediaUrl));
+            } else {
+                contents.add(new Audio(mediaUrl));
```
Comment (paulodiniz) [inline lib/java/mapping/src/main/java/co/airy/mapping/sources/twilio/TwilioMapper.java:40]: Twilio doesn't allow text  + media (except images) 😂

File: lib/java/mapping/src/test/java/co/airy/mapping/TwilioTest.java [hunk 2]
```diff
@@ -40,7 +42,24 @@ void canRenderImage() throws Exception {
                 "&MediaUrl=" + imageUrl;
 
         final List<Content> message = mapper.render(event);
+        assertTrue(message.stream().anyMatch(c -> c instanceof Text));
```
Comment (paulodiniz) [inline lib/java/mapping/src/test/java/co/airy/mapping/TwilioTest.java:45]: Asserting that `Text` is still included when sending images

File: lib/java/mapping/src/test/java/co/airy/mapping/TwilioTest.java [hunk 3]
```diff
@@ -40,7 +42,24 @@ void canRenderImage() throws Exception {
...
+        final String body = "Heres an audio!";
+        final String audioUrl = "https://demo.twilio.com/owl.mp3";
+
+        String event = "ApiVersion=2010-04-01&SmsSid=SMbc31b6419de618d65076200c54676476&SmsStatus=received" +
+                "&SmsMessageSid=SMbc31b6419de618d65076200c54676476&NumSegments=1&To=whatsapp%3A%2B" +
+                "&From=whatsapp%3A%2B&MessageSid=SMbc31b6419de618d65076200c54676476" +
+                "&MediaUrl=" + audioUrl;
+        final List<Content> message = mapper.render(event);
+
+        assertTrue(message.stream().filter(c -> c instanceof Text).findAny().isEmpty());
```
Comment (paulodiniz) [inline lib/java/mapping/src/test/java/co/airy/mapping/TwilioTest.java:61]: Asserting that `Text` is *not* included when sending audios

**Review by lucapette — APPROVED** [review]
:shipit:

**Pull request merged**